app/views/foreman_discovery/debian_kexec.erb

```diff
@@ -12,12 +12,15 @@ environments. The template must generate JSON format with the following items
 "kernel", "initram", "append" and "extra". The kexec command is composed in
 the following way:
 
-kexec --force --debug --append=$append --initrd=$initram $extra $kernel
+kexec --force --reset-vga --append=$append --initrd=$initram $extra $kernel
 
 Please read kexec(8) man page for more information about semantics.
-Extra options like --reset-vga can be set via "extra" array.
```

Comment:
This is no longer true, we've had `--reset-vga` for few releases but we had to remove this by default because it was causing some problems on some systems. So it's really without `--reset-vga` by default, if you want it then add it to this template.